services/notification/notifiers/mixins/message/sections.py

```diff
@@ -105,10 +105,16 @@ async def do_write_section(self, comparison, diff, changes, links, behind_by=Non
 class NewHeaderSectionWriter(BaseSectionWriter):
     def _possibly_include_test_result_setup_confirmation(self, comparison):
         if comparison.all_tests_passed():
-            yield ("")
-            yield (
-                ":white_check_mark: All tests successful. No failed tests found :relaxed:"
-            )
+            if comparison.test_results_error():
```

Comment:
So... if there are no tests then we assume all tests passed? Crazy... Can we fix that? Maybe by checking that there's at least 1 test in the "all tests passed".
I didn't realize this was a possibility before 😅